src/bot/.env.example

```diff
@@ -0,0 +1,2 @@
+TOKEN=
+API_URL=
```

Comment:
I would prefer for this to be in a README

Reply:
I will put environment variable instructions in the readme _**as well**_ , but I think it would also be nice to have a .env.example file(or some other name) as well within the repository because that is a standard in the development community.

Reply:
if that is a standard then we can keep it, I am not used to contributing projects. 
Thanks for letting me know.

Reply:
I've made the necessary changes to the README.md file. Please check it out whenever you get a chance.